lib/Doctrine/ORM/Configuration.php

```diff
@@ -545,6 +549,8 @@ public function addCustomDatetimeFunction($name, $className)
      * @param string $name
      *
      * @return string|null
+     * 
```

Comment:
That kind of trailing whitespace makes the build fail. Please run `vendor/bin/phpcbf` to fix that